CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
 
 ### Fixed
 
+- crud tests with Tarantool 3.0 (#293)
```

Comment:
I'm not sure test fixes should be covered by a changelog entry since it's not a user-visible change

Reply:
I usually add entries like this for the repository:

This is a low-level library, users could look at tests and run them.